api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -53,6 +53,10 @@ type ClientTrafficPolicySpec struct {
 	//
 	// +optional
 	TCPKeepalive *TCPKeepalive `json:"tcpKeepalive,omitempty"`
+	// EnableHTTP3 enables HTTP/3 support on the listener.
+	// Disabled by default.
+	// +optional
+	EnableHTTP3 bool `json:"enableHTTP3,omitempty"`
```

Comment:
lets make this `*bool` since its optional

Reply:
I would love to just have a enable knob, but agree with @LanceEa, an empty struct maybe a good call for this API because some user may want to decide the listening port, and everyone loves tweaking port values to adhere with corp perimeter policies